recipe/bld.bat

```diff
@@ -36,6 +17,7 @@ cmake -GNinja ^
     -D ENABLE_ZLIB_COMPRESSION=ON ^
     -D BUILD_EXAMPLES=OFF ^
     -D BUILD_TESTING=OFF ^
+    -D CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=ON ^
```

Comment:
I don't think we should be doing this as they have a set of public symbols and this would export all symbols.

Can't we use VS generator?

Reply:
Then we're back to the MSBuild-not-found issue on Azure.

Reply:
That issue was fixed when we switched to VS2017